### ISSUE TITLE
[Question] Is it possible to debug clones?

### ISSUE THREAD

**Issue by Fr1ar** [VeriorPies/ParrelSync#73]
After I cloned my project in the Clones Manager and attached Visual Studio to Unity it turned out that I can't debug the second instance.
All breakpoints are disabled:

![image](https://user-images.githubusercontent.com/11349335/154834513-0fae774c-257e-42a1-8c85-1b294232926b.png)

In the first instance everything works well.
What can I do with this?

**Comment by kojonek2** [comment]
@Fr1ar I can debug clones without a problem. Be sure to generate new solution in the clone folder (by double-clicking some script in clone unity) and then attach debuger in that new solution

**Comment by Fr1ar** [comment]
Yes, I do exactly what you say, but breakpoints still don't work.
